packages/expo-localization/src/Localization.types.ts

```diff
@@ -65,3 +65,134 @@ export type Localization = {
    */
   timezone: string;
 };
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code.
+   * @example `'en-US'`, `'es-419'`, `'pl-PL'`.
+   */
+  languageTag: string;
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) without the region code.
+   * @example `'en'`, `'es'`, `'pl'`.
+   */
+  languageCode: string;
+  /**
+   * The region code for your device that comes from the Region setting under Language & Region on iOS, Region settings on Android and is parsed from locale on Web (can be `null` on Web).
+   */
+  regionCode: string | null;
+  /**
+   * Currency code for the locale.
+   * @example `'USD'`, `'EUR'`, `'PLN'`.
+   * Returns `null` on Web, use a table lookup based on region instead.
+   */
+  currencyCode: string | null;
+  /**
+   * Currency symbol for the locale.
+   * @example `'$'`, `'€'`, `'zł'`.
+   * Returns `null` on Web, use a table lookup based on region (if available) instead.
+   */
+  currencySymbol: string | null;
+  /**
+   * Decimal separator used for formatting numbers with fractional parts.
+   * @example `'.'`, `','`.
+   */
+  decimalSeparator: string | null;
+  /**
+   * Digit grouping separator used for formatting large numbers.
+   * @example `'.'`, `','`.
+   */
+  digitGroupingSeparator: string | null;
+  /**
+   * Text direction for the locale. One of: `'ltr'`, `'rtl'`, but can also be `null` on older browsers without support for the [textInfo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/textInfo) property in [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) API.
```

Comment:
```suggestion
   * Text direction for the locale. One of: `'ltr'`, `'rtl'`, but can also be `null` on some browsers without support for the [textInfo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/textInfo) property in [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) API.
```

Just a wording tweak, but according to docs, FF doesn't support that property even in the latest versions.

Reply:
Agreed, that wording was a bit too optimistic :/ there's an issue on bugzilla that seems stuck for that support